glossary/glossary.md

```diff
@@ -0,0 +1,57 @@
+# Multimedia 
++ #### *Packet* is a formatted unit of data transmitted over network 
++ #### *Frame* is a basic data unit used by media coding formats
```

Comment:
I would also mention that this term can relate to a video frame, and add some examples [from here](https://en.wikipedia.org/wiki/Video_compression_picture_types)